tools/page-status/diff.html

```diff
@@ -0,0 +1,76 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta
+    http-equiv="Content-Security-Policy"
+    content="script-src 'nonce-aem' 'strict-dynamic'; base-uri 'self'; object-src 'none';"
+    move-to-http-header="true"
+  >
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta name="lab" content="true">
+  <title>Page Diff Viewer</title>
+  <script nonce="aem" src="/scripts/aem.js" type="module"></script>
+  <script nonce="aem" src="/scripts/scripts.js" type="module"></script>
+  <link rel="stylesheet" href="/styles/styles.css" />
+  <script nonce="aem" src="./diff.js" type="module" defer></script>
+  <link rel="stylesheet" href="./diff.css" />
+</head>
+<body class="page-diff">
+  <header></header>
+  <main>
+    <div>
```

Comment:
While testing locally, I noticed that `diff-info`, `diff-loading`, diff-no-results`, `diff-error`, `diff-results` is getting run through aem.js and getting processed as blocks (because of how markup is structured and the class names. 

It's causing nuisance errors in browser console like below

<img width="1698" height="511" alt="Image" src="https://github.com/user-attachments/assets/0474c1c1-ed18-473a-9061-9e83a87050df" />

Suggesting a fix in the next comment..

Reply:
good point. will take a look